src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseApplicationResponseController.java

```diff
@@ -48,13 +48,14 @@ public Document generateC7DraftDocument(
         @PathVariable("caseId") String caseId,
         @PathVariable("partyId") String partyId,
         @RequestHeader(HttpHeaders.AUTHORIZATION) @Parameter(hidden = true) String authorisation,
+        @RequestHeader("isWelsh") boolean isWelsh,
         @RequestHeader("serviceAuthorization") String s2sToken) throws Exception {
 
         CaseDetails caseDetails = coreCaseDataApi.getCase(authorisation, s2sToken, caseId);
         CaseData caseData = CaseUtils.getCaseData(caseDetails, objectMapper);
 
         caseApplicationResponseService.updateCurrentRespondent(caseData, YesOrNo.Yes, partyId);
-        return caseApplicationResponseService.generateC7DraftDocument(authorisation, caseData);
+        return caseApplicationResponseService.generateC7DraftDocument(authorisation, caseData,isWelsh);
```

Comment:
format please. also dont think its needed

Reply:
formatted text  and  as discussed this flag is required to generate draft C7